backend/src/apiserver/server/job_server.go

```diff
@@ -90,22 +89,19 @@ func (s *JobServer) createJob(ctx context.Context, job *model.Job) (*model.Job,
 	if err != nil {
 		return nil, util.Wrapf(err, "Failed to create a run due to invalid experimentId and namespace combination")
 	}
+	if common.IsMultiUserMode() && namespace == "" {
```

Comment:
Shall we also verify that `namespace == ""` in single-user mode?

Reply:
In a single-user mode, we simply skip authorization. So no check are needed.